package.json

```diff
@@ -17,16 +17,16 @@
     "test": "jest --maxWorkers=4"
   },
   "engines": {
-    "node": ">=8"
+    "node": ">=18"
   },
   "devDependencies": {
     "eslint": "^4.18.2",
     "eslint-plugin-node": "^6.0.1",
-    "husky": "^0.14.3",
-    "jest": "^25.0.0",
+    "husky": "^8",
+    "jest": "^27.0.0",
     "lerna": "^2.9.0",
     "lint-staged": "^9.4.2",
```

Comment:
We have updated lint-staged to v15 on other dependencies, any reason not to here?